### PR TITLE
Add a script so that enabling cli debugging is easier/automated

### PR DESCRIPTION
**Pull request by singularo**
Making things easier

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,8 @@ RUN gem install bundler
 
 # Install Composer, Drupal Console and Drush.
 RUN wget -q https://getcomposer.org/installer -O - | php -- --install-dir=/usr/local/bin --filename=composer \
-&& wget https://drupalconsole.com/installer -O /usr/local/bin/drupal && chmod +x /usr/local/bin/drupal && /usr/local/bin/drupal init \
+&& wget https://drupalconsole.com/installer -O /usr/local/bin/drupal \
```
Comment (pingers) [inline Dockerfile:38]: Guessing this was a one liner to keep all the related things together, but meh.

Comment (singularo) [reply Dockerfile:38]: Yeah, don't really care about that, jus taccidentally slipped it in ;-)

**Comment by pingers** [comment]
I still feel like 9001 is a crap experience as a new dev... but guess we'll live with that.

Looks great otherwise!

**Comment by singularo** [comment]
We can change it to 9000, but thats the default port for php-fpm.. I guess it depends if we'll ever switch to nginx+php-fpm..

**Pull request closed without merging**